cmd/cli/app/log_info.go

```diff
@@ -42,6 +51,47 @@ var logInfoCmd = &cobra.Command{
 
 		logInfo := result.GetPayload()
 		fmt.Printf("Tree Size: %v, Root Hash: %v\n", *logInfo.TreeSize, *logInfo.RootHash)
+
+		keyHint, err := base64.StdEncoding.DecodeString(logInfo.SignedTreeHead.KeyHint.String())
+		if err != nil {
+			log.Fatal(err)
+		}
+		logRoot, err := base64.StdEncoding.DecodeString(logInfo.SignedTreeHead.LogRoot.String())
+		if err != nil {
+			log.Fatal(err)
+		}
+		signature, err := base64.StdEncoding.DecodeString(logInfo.SignedTreeHead.Signature.String())
+		if err != nil {
+			log.Fatal(err)
+		}
+		sth := trillian.SignedLogRoot{
+			KeyHint:          keyHint,
+			LogRoot:          logRoot,
+			LogRootSignature: signature,
+		}
+
+		// validate result is signed with the key we're aware of
+		keyResp, err := rekorClient.Tlog.GetPublicKey(nil)
```

Comment:
This is probably a larger question - but do you think we should store this public key as well? It shouldn't change, really ever.

Reply:
I thought about throwing it into the Rekor config file client side, which would be a nice thing to do IMO.

Reply:
That sounds like a good approach, this way developers machines can be locked down (to degree), so they only use trusted / nominated public keys.

Reply:
Right, it's sort of like SSH's TOFU model. I don't think it can be rotated without invalidating the entire t-log, so it should basically never change.

Reply:
Feel free to leave as a TODO though.